### PR TITLE
refactor: cleanup and organse cli commands

### PR DESCRIPTION
**Pull request by ravisuhag**
- Organise cli commands 
- Unified interface to run recipe for file and directory.

### CODE REVIEW

File: cmd/run/run.go [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+}
+
+// Run a single recipe
+func run(recipeFile string) {
+	c, err := config.LoadConfig()
+	if err != nil {
+		log.Fatal(err)
+	}
+	log := logger.New(c.LogLevel)
+	plugins.Log.Level = log.Level
```
Comment (StewartJingga) [inline cmd/run/run.go:48]: should we hide this Log level update behind a function?
e.g.
```
plugins.SetLogLevel(log.Level)
```

File: cmd/run/run.go [hunk 2]
```diff
@@ -0,0 +1,83 @@
+package run
```
Comment (StewartJingga) [inline cmd/run/run.go:1]: having a package for a command should be fine, but I was just thinking if it is an overkill 🤔 

what is the reason for a command to have its own package?

File: cmd/run/run.go (hunk 1 above)
Comment (StewartJingga) [reply cmd/run/run.go:48]: or maybe we can even pass the logger to the plugins so we would be using the same instance.
e.g.
```
plugins.SetLog(log)
```

Comment (ravisuhag) [reply cmd/run/run.go:48]: Yes, we can do that. Also, we are using native logger and logrus both. Didn't change it this PR but we should use just one of them.

File: cmd/run/run.go (hunk 2 above)
Comment (ravisuhag) [reply cmd/run/run.go:1]: It allows easy navigation of commands. When the commands within commands will grow. As an ex. gen will multiple of these.

File: cmd/run/run.go (hunk 1 above)
Comment (StewartJingga) [reply cmd/run/run.go]: oh thought it was changed this PR, tried my proposed changes in my local and had some issues due to how we register our plugins now.

let's keep it this way first then.

File: cmd/run.go [hunk 3]
```diff
@@ -22,8 +56,56 @@ func run(recipeFile string) {
 	if err != nil {
 		log.Fatal(err)
 	}
+	fmt.Println(rcp)
```
Comment (StewartJingga) [inline cmd/run.go:59]: is this intended?

Comment (ravisuhag) [reply cmd/run.go]: No, removed.

**Review by StewartJingga — APPROVED** [review]

**Pull request merged**